docker/r10k/docker-entrypoint.d/10-analytics.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ "${PUPPERWARE_DISABLE_ANALYTICS}" = "true" ]; then
-    echo "($0) Pupperware analytics disabled; skipping metric submission"
+if [ "${PUPPERWARE_ANALYTICS_ENABLED}" != "true" ]; then
+    echo "($0) Pupperware analytics not enabled; skipping metric submission"
```

Comment:
We _could_ add another message here saying how to enable metrics 🤷‍♂️ 

    echo "($0) Set PUPPERWARE_ANALYTICS_ENABLED=true to submit metrics"

Reply:
Nope, actually I'm going to remove all printouts from the container since this is a CLI, not a service.

These container messages being printed out on every run of the CLI is a poor experience.